docs/src/user-guide/guides-retention.md

```
@@ -0,0 +1,167 @@
+## Configuration
+CLP allows users to configure different **retention_periods** for different types of data. 
+Additionally, a custom **sweep_interval** can be set for each type of data, which determines how
+frequently its garbage collection job is executed. These settings can be configured in 
+`etc/clp-config.yml`. 
+
+### Configure retention period
+To configure a retention period, update the appropriate `.retention_period` key in
+`etc/clp-config.yml` with the desired retention period in minutes.
+
+For example, to configure an archive retention period of 30 days (43,200 minutes):
+```yaml
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)